sda/processor.go

```diff
@@ -95,8 +93,7 @@ func (p *ServiceProcessor) RegisterMessages(procs ...interface{}) error {
 	return nil
 }
 
-// Process implements the Processor interface and dispatches ClientRequest message
-// and InterServiceMessage.
+// Process implements the Processor interface and dispatches ClientRequest messages.
 func (p *ServiceProcessor) Process(packet *network.Packet) {
```

Comment:
Ok we have panic now and I'm opnening... an issue for this weird message dispatching that we have now. :D